src/app/user/profile/edit-password-modal/edit-password-modal.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Router } from '@angular/router';
+import { ModalController } from '@ionic/angular';
+import { Subscription } from 'rxjs';
+import { AuthService } from 'src/shared/services/auth.service';
+
+@Component({
+  selector: 'app-edit-password-modal',
+  templateUrl: './edit-password-modal.component.html',
+  styleUrls: ['./edit-password-modal.component.scss'],
+})
+export class EditPasswordModalComponent implements OnInit {
+  editSubscription$?: Subscription;
```

Comment:
editPassword maybe ? (In edit-email-modal is named editEmail). Or editPassSubscription and editEmailSubscription ?